Formula/futhark.rb

```diff
@@ -22,7 +22,7 @@ class Futhark < Formula
 
   def install
     system "cabal", "v2-update"
-    system "cabal", "v2-install", *std_cabal_v2_args
+    system "cabal", "v2-install", *std_cabal_v2_args, "--constraint=bytestring==0.10.10.1"
```

Comment:
Would this need to be updated with version bumps? If so, a comment would be good the explains how.